pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj

```diff
@@ -85,28 +90,6 @@
     </ProjectReference>
   </ItemGroup>
   
-  <ItemGroup Condition="'$(PackageTargetRuntime)' != ''">
-    <ArchitectureSpecificNativeFile Include="$(DotNetHostBinDir)/dotnet$(ApplicationFileExtension)" />
```

Comment:
The same goes for

LibraryFilePrefix
LibraryFileExtension
SymbolFileExtension

---
In reply to: [106176307](https://github.com/dotnet/core-setup/pull/1728#discussion_r106176307) [](ancestors = 106176307)